core/arch/arm/plat-imx/conf.mk

```diff
@@ -3,27 +3,23 @@ include core/arch/$(ARCH)/plat-$(PLATFORM)/platform_flags.mk
 CROSS_COMPILE	?= arm-linux-gnueabihf-
 COMPILER	?= gcc
 
-CFG_ARM32_core ?= y
-CFG_MMU_V7_TTB ?= y
-
-core-platform-cppflags	 = -I$(arch-dir)/include
+core-platform-cppflags  = -I$(arch-dir)/include
+core-platform-cppflags += -D_USE_SLAPORT_LIB
+core-platform-cppflags += -DCFG_NO_TA_HASH_SIGN
```

Comment:
It's OK, I can do it in this PR.
`plat-sunxi` also has this `-D_USE_SLAPORT_LIB`, should it be removed too? Anyone knows the purpose of this define?